src/core/truthTable/pla_parser.cpp

```diff
@@ -0,0 +1,91 @@
+#include "core/truthTable/pla_parser.hpp"
+
+namespace syrec {
+
+    [[maybe_unused]] void pla_parser(std::istream& in, syrec::TruthTable& reader) {
+        std::string line;
+        std::regex  whitespace("\\s+");
+
+        while (in.good() && getline(in, line)) {
+            trim(line);
+            line = std::regex_replace(line, whitespace, std::string(" "));
+            if ((line.empty()) || (line.rfind('#', 0) == 0) || (line.rfind(".i", 0) == 0) || (line.rfind(".o", 0) == 0) || (line.rfind(".ilb", 0) == 0) || (line.rfind(".ob", 0) == 0) || (line.rfind(".p", 0) == 0) || (line.rfind(".type ", 0) == 0)) { continue; }
+
+            //std::string comment( std::find_if( line.begin(), line.end(), []( char c ) { return c != '#'; } ) );
+            //reader.on_comment( comment );
+
+            //reader.on_num_inputs( static_cast<unsigned>( line.substr( 3 ) ) );
+
+            //reader.on_num_outputs( boost::lexical_cast<unsigned>( line.substr( 3 ) ) );
+
+            //reader.on_num_products( boost::lexical_cast<unsigned>( line.substr( 3 ) ) );
+
+            //reader.on_type( line.substr( 6 ) );
+            else if (line == ".e") {
+                break;
+                //reader.on_end();
+            }
+
+            else {
+                assert(!line.empty() && (line[0] == '0' || line[0] == '1' || line[0] == '-'));
```

Comment:
this does not handle the tilde character that is handled in the other parser function.